app/controllers/users_controller.rb

```diff
@@ -9,6 +9,15 @@ def index
   def show
   end
 
+  def toggle_admin
+    @user.admin = !@user.admin
+    if @user.save
+      redirect_to user_path(@user), notice: 'Admin status updated'
+    else
+      redirect_to user_path(@user), notice: 'Unable to update admin status'
```

Comment:
This is the line that isn't covered, I'm wondering if there isn't any validation happening here do we need this?  Is there a scenario where it wouldn't save?

Reply:
Yeah, this is tricky - if you can't manipulate a condition where it would fail for a test then perhaps it's overly defensive and should be removed

Reply:
Removed